packages/helpermodules/utils/processing_counter.py

```diff
@@ -0,0 +1,26 @@
+import threading
```

Comment:
```suggestion
from threading import Lock, Event
```
Ich hatte die Imports in anderen Dateien vor Kurzem etwas abgespeckt. Sollte hier dann auch gemacht werden. `threading` ist einrecht großen Modul.